caikit_huggingface_demo/runtime/config/config.yml

```diff
@@ -2,7 +2,7 @@ runtime:
   # The runtime library (or libraries) whose models we want to serve using Caikit Runtime. This should
   # be a snake case string, e.g., caikit_nlp or caikit_cv.
   library: runtime
-  local_models_dir: models
+  local_models_dir: example_models
```

Comment:
Couple things:
1 - Need some agreement on which models to load by default ("we" had agreed on the one)
2 - Need to decide whether to move those into "models" or do this.
Either way I don't mind this once we agree (defaults vs extras?) but this change makes the README.md out-of-date.  As you have noticed it gets confusing.  So even this one-word change breaks the README in several places.